src/include/compat/wctype.h

```diff
@@ -44,7 +44,7 @@
 #define iswspace  ucd_isspace
 #define iswupper  ucd_isupper
 #define iswxdigit ucd_isxdigit
-#define tolower ucd_tolower
-#define toupper udc_toupper
+// #define tolower ucd_tolower
```

Comment:
If I don't uncomment them, it causes errors in my code since I am using `std::tolower` and `std::toupper` in my code.